### DISCUSSION TITLE
Is it possible to disable logs in dev console?

### DISCUSSION THREAD

**Discussion by baymac** [DuCanhGH/next-pwa#110] [Q&A]
I could not find a config to disable the logs, which makes it difficult to debug with custom logs.

<img width="215" alt="Screenshot 2023-10-20 at 12 14 44 PM" src="https://github.com/DuCanhGH/next-pwa/assets/23079344/b0c05459-1bc1-4a9b-8e11-749d107f3afc">

**Comment by DuCanhGH** [comment]
@baymac add `worker/index.ts`:

```ts
declare const self: ServiceWorkerGlobalScope;

// https://developer.chrome.com/docs/workbox/troubleshooting-and-logging/#turn-off-logging-in-development-builds-in-any-workflow
self.__WB_DISABLE_DEV_LOGS = true
```
